less/Services2.less

```diff
@@ -0,0 +1,159 @@
+* {
+    box-sizing: border-box;
+  }
+
+  @import 'mixins.less';
+  
+@import 'tabs.less';
+  
+.home {
+    .content-section-top{
+        
+        .headerimg{
+            width: 1000px;
+            height: 175px;
+        }
+        @media @phone {
+            
+        .pool{
+            display:none;
+        }
+        
+    }
+    h2{
+       color: @header-font;
+    }
+    p{
+      color: @main-font-color;
+    }
+    }
+
+    .text-content-top{
+      
+      margin-top: 20px;
+    
+        display: flex;
+        flex-direction: row;
+        
+      justify-content: space-between;
+      h2{
+        display: flex;
+        flex-direction: column;
+      }
+      p{
+        display: flex;
+        flex-wrap: wrap;
+      justify-content: space-between;
+      }
+      .btn{
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+      }
+      
+      img{
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        margin-bottom: 30px;
+        
+        
+}
+.sketch{
+    @media @phone {
+        width: 100%
+  }
+}
+    }
+    
+      
+
+
+    .big-contain {
+      margin: 30px 0;
+      
+  
+     
+    
+      .text-content {
+        width: 48%;
+        padding-right: 1%;
+        display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+        .new{
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: space-between;
+        }
+
+        
+      }
+  
+      
+      }
+    
+  
+    .inverse-content {
+      padding-bottom: 30px;
+      padding-top: 40px;
+      border-top: 1px solid @navigation-border;
+      border-bottom: 1px solid @navigation-border;
+      p{
+        color: @main-font-color;
+      }
+  
+      .text-content {
+        padding-left: 1%;
+        padding-right: 0;
+      }
+  
+      .img-content {
+        padding-right: 1%;
+        padding-left: 0;
+        float: right 10px;
+      }
+    }
+  
+    .content-destination {
+      width: 75%;
+      margin: 0 auto 30px;
+  
+      img {
+        border-radius: 10px;
+      }
+    }
+  
+    .content-pick {
+      padding-top: 30px;
+      border-top: 2px dashed @navigation-border;
+      display: flex;
+      justify-content: space-between;
+      
+      @media @phone {
+        display: flex;
+        flex-wrap: wrap;
+      }
+      
+      .walk {
+        width: 30%;
+        margin-bottom: 30px;
+        @media @phone {
+          width: 100%;
+          
+        
+        }
+        
+        
```

Comment:
Try to find an extension to manage your indentation for you, it's very easy to get confused about the nesting relationships if the indentation is off.